pci.h

```diff
@@ -60,6 +60,7 @@ class PciHandle
     PciHandle(uint32 groupnr_, uint32 bus_, uint32 device_, uint32 function_);
 
     static bool exists(uint32 bus_, uint32 device_, uint32 function_);
+    static int openMcfgTable();
```

Comment:
please make this function protected

Reply:
Method is defined in `PciHandle` but used from `PciHandleM`/`PciHandleMM`. Protected will not compile.

Reply:
did you consider making PciHandleM / PciHandleMM friends of PciHandle? I just dont like to expose the low level openMcfgTable function as a part of the public interface of PciHandle

Reply:
Yep, that will do. Thanks!